src/nxt_main.h

```diff
@@ -74,6 +74,7 @@ typedef struct nxt_sockaddr_s           nxt_sockaddr_t;
 #include <nxt_malloc.h>
 #include <nxt_mem_map.h>
 #include <nxt_socket.h>
+#include <nxt_socket_msg.h>
```

Comment:
Suggest do not include nxt_socket_msg.h in nxt_main.h since it is explicitly included in files where it required.

Reply:
Ok, but then I need to include it also in `nxt_port_socket.c`.